core/src/HIP/Kokkos_HIP_BlockSize_Deduction.hpp

```diff
@@ -169,14 +169,8 @@ struct HIPGetMaxBlockSize<DriverType, Kokkos::LaunchBounds<>, true> {
                             size_t const vector_length,
                             size_t const shmem_extra_block,
                             size_t const shmem_extra_thread) {
-// FIXME_HIP -- remove this once the API change becomes mature
-#if !defined(__HIP__)
-    using blocktype = unsigned int;
-#else
-    using blocktype = int;
-#endif
-    blocktype numBlocks = 0;
-    int blockSize       = 1024;
+    int numBlocks = 0;
+    int blockSize = LaunchBounds::maxTperB;
```

Comment:
This doesn't currently compile. I guess you want to keep it as 1024 for this pull request.

Reply:
Ahh, sorry -- we'll want to make sure this gets merged before !3299 so that the end result has `LaunchBounds::maxTperB` then